sdk/purview/azure-purview-sharing/swagger/README.md

```diff
@@ -0,0 +1,45 @@
+# Azure Purview for Python
+
+> see https://aka.ms/autorest
+
+### Setup
+
+Install Autorest v3
+
+```ps
+npm install -g autorest
+```
+
+### Generation
+
+```ps
+cd <swagger-folder>
+autorest
+```
+
+### Settings
+
+```yaml
+package-name: "@azure-rest/purview-sharing"
+title: Purview Sharing Client
+description: Purview Sharing Client
+generate-metadata: false
+license-header: MICROSOFT_MIT_NO_VERSION
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2023-02-15-preview/share.json
+output-folder: ../azure/purview/sharing
+namespace: azure.purview.sharing
+clear-output-folder: true
+no-namespace-folders: true
+python: true
+package-version: 1.0.0b1
+rest-level-client: true
+add-credential: true
+credential-scopes: "https://purview.azure.net/.default"
+generate-test: false
+generate-samples: true
+modelerfour:
+  lenient-model-deduplication: true
+only-path-params-positional: true
+version-tolerant: true
+directive: rename-operation:[ from:"SentShares_Get", to:"SentShares_GetSentShare", from:"SentShares_Create", to:"SentShares_CreateOrReplaceSentShare", from:"SentShares_Delete", to:"SentShares_DeleteSentShare", from:"SentShares_List", to:"SentShares_GetAllSentShares", from:"SentShares_GetInvitation", to:"SentShares_GetSentShareInvitation", from:"SentShares_CreateInvitation", to:"SentShares_CreateSentShareInvitation", from:"SentShares_DeleteInvitation", to:"SentShares_DeleteSentShareInvitation", from:"SentShares_ListInvitations", to:"SentShares_GetAllSentShareInvitations", from:"SentShares_NotifyUserInvitation", to:"SentShares_NotifyUserSentShareInvitation", from:"ReceivedShares_Get", to:"ReceivedShares_GetReceivedShare", from:"ReceivedShares_Create", to:"ReceivedShares_CreateOrReplaceReceivedShare", from:"ReceivedShares_Delete", to:"ReceivedShares_DeleteReceivedShare", from:"ReceivedShares_ListAttached", to:"ReceivedShares_GetAllAttachedReceivedShares", from:"ReceivedShares_ListDetached", to:"ReceivedShares_GetAllDetachedReceivedShares", ]
```

Comment:
Why don't change swagger directly so that all languages can be benefied?

Reply:
When it comes to operation ID, I would have just edited the Swagger directly. What @msyyc is that we need to have those rules duplicated everywhere, while we can just have a more accurate Swagger. OperationID is not a concept that exists on the server, so it's not considered a breaking change to update it as long as the SDK is not shipped yet. I know C# is out, but if you used the same rule in the C# SDK, moving them from directive to the Swagger will be a no-op on the C# SDK.

Reply:
When you say swagger, you mean here in readme.md for swagger? because this is what was done here and in JS, Java and C# SDKs

Reply:
Got your point, you mean the attributes for operation id in C# attributes. The team went for JS, Java and C# and Python. We used the same directives. Thanks for pointing out. Marking as resolved.